meta-oe/recipes-support/serial/serial_1.2.1.bb

```diff
@@ -0,0 +1,26 @@
+SUMMARY = "Cross-platform library for interfacing with rs-232 serial like ports"
+HOMEPAGE = "http://wjwwood.io/serial/"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://README.md;beginline=53;endline=62;md5=049c68d559533f90250404746e6a1045"
+
+SRC_URI = " \
+    git://github.com/wjwwood/serial.git;nobranch=1;tag=${PV} \
```

Comment:
We should not use `tags` in SRC_URI since they are considered floating, please use SRCREV